README.md

```diff
@@ -178,6 +178,50 @@ itWillReturn
 ```
 ````
 
+### `:playground`
+
+Ank also has support to turn Kotlin snippets into Kotlin Playgrounds w/ support for Arrow. 
+For a snippet to be runnable it needs to contain a `main` function,
+and you can mark the begin and end of the example using `//sampleStart` & `//sampleEnd`.
```

Comment:
I would add a reminder about the need for having all the sources you need in the snippet otherwise it'll fail. Also clarify that if you need to showcase compile time generated syntax you'll need to copy those generated elements on the snippet, given Playgrounds can only depend on the sources in the snippet, and the playgrounds server doesn't run kapt either.

About the different behavior between Ank and playgrounds when it comes to classpath dependencies:  Playgrounds can also work incrementally so you don't need to redeclare everything on every single snippet, like Ank does. You can do it with an "incremental" flag but last time I tried random stuff failed, so I'm not completely sure whether that works fine.